atk.env

```diff
@@ -1,4 +1,4 @@
 ROS_DISTRO=humble
 
-DEFAULT_APT_DEPENDENCIES=" bash vim git git-lfs python3-pip"
-DEFAULT_PIP_REQUIREMENTS=" setuptools==58.2.0 "
+DEFAULT_APT_DEPENDENCIES=" bash vim git git-lfs python3-pip "
```

Comment:
why is this space needed?

Reply:
idk more of a future proofing. probably not needed, but makes this valid: `"${DEFAULT_APT_DEPENDENCES}another_dependency". can remove if you want.